### DISCUSSION TITLE
can't use the annotation

### DISCUSSION THREAD

**Discussion by norenellove** [sumatrapdfreader/sumatrapdf#3169] [Q&A]
Just downloaded sumatra today.
but i can't find annotation in the tooltip
and pressing a doesn't work
help plz～～～
I've already checked similar discussions But still didn't solve my problem...😢

**Comment by GitHubRulesOK** [comment]
you should have downloaded either
 recent 3.4.6 current https://www.sumatrapdfreader.org/download-free-pdf-viewer
or 3.5 pre release https://www.sumatrapdfreader.org/prerelease
so check your help about for  version number

**Reply by norenellove** [reply]
Thank you for replying. But I'm pretty sure it's the latest version.

**Comment by tactilis** [comment]
@norenellove 
> I'm pretty sure it's the latest version

Please don't be "pretty sure".

Could you look at menu `Help | About` and report your exact version number please.

![image](https://user-images.githubusercontent.com/4060092/206874604-2177aa13-761b-49f6-90d1-ff10f2724649.png)

**Comment by norenellove** [comment]
ok. let me show you.

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by tactilis** [comment]
@norenellove 
> ok. let me show you.
> 获取Outlook for Android<https://aka.ms/AAb9ysg>

That link just leads to this page:

![image](https://user-images.githubusercontent.com/4060092/206929586-884015e2-45c1-40b9-8c68-2ee640decd0e.png)
 
 
Could you not just say what version `Help | About` shows please?

Or paste a screenshot here [https://github.com/sumatrapdfreader/sumatrapdf/discussions/3169](https://github.com/sumatrapdfreader/sumatrapdf/discussions/3169).

Note that if you reply by email, your screenshots are not shown.

**Reply by norenellove** [reply]
<img width="914" alt="屏幕截图 2022-12-12 175252" src="https://user-images.githubusercontent.com/120198723/207015383-8aa0aae7-2812-47f1-ae97-c97039c419d4.png">

**Reply by norenellove** [reply]
<img width="472" alt="image" src="https://user-images.githubusercontent.com/120198723/207015548-a37e7d85-6185-42d2-80de-27cecfa9e40a.png">

**Reply by GitHubRulesOK** [reply]
annotation is only for PDF format 
if that is eBook then annotation **no longer** works there. You need a different app to annotate plain text or other fiction/comicbook filetypes.

**Comment by norenellove** [comment]
oh,ok, i see. Truly, it's mobi.
Thanks!

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
